internal/gatewayapi/testdata/backendtlspolicy-serviceimport-target.in.yaml

```diff
@@ -82,7 +82,7 @@ endpointSlices:
       name: service-import-1
       namespace: default
       labels:
-        kubernetes.io/service-name: service-import-1
+        multicluster.kubernetes.io/service-name: service-import-1
```

Comment:
This test was missing correct `multicluster.kubernetes.io/service-name` label for the EndpointSlice.